### PR TITLE
Display copy + launch errors

### LINKED ISSUE #522: Missing Instance Limit Warning when Creating Tale Copy
The user doesn't see an error message when doing a copy + launch when there are two instances running.

To Reproduce:
1. Make sure you have two instances running
2. Launch a Tale that you don't own
3. Note that you don't receive an error about the instance limit

### PR DESCRIPTION
**Pull request by ThomasThelen**
This PR adds a modal dialog that opens when copy + launching fails. This fixes the issue where the user wasn't getting a notification that they already had two instances running. Fixes #522 

WGdUlAuBXw

To Test:
1. Deploy this branch
2. Launch two instances
3. Attempt to copy + launch a Tale you don't own
4. See the error notification

### CODE REVIEW

File: app/components/ui/copy-on-launch-modal/component.js [hunk 1]
```diff
@@ -32,11 +29,9 @@ export default Component.extend({
           if (originalTale) {
               
               let handleLaunchError = (tale, err) => {
-                // deal with the failure here
-                //tale.set('launchStatus', 'error');
-                //tale.set('launchError', err.message || err);
-                
                 console.error('Failed to launch Tale', err);
+                self.set("errorMessage", err.message);
+                $('.ui.modal.compose-error').modal('show');
```
Comment (bodom0015) [inline app/components/ui/copy-on-launch-modal/component.js]: Possibly a copy-paste error here. I think it should be:
```suggestion
                $('.ui.modal.copy-error').modal('show');
```

Comment (ThomasThelen) [reply app/components/ui/copy-on-launch-modal/component.js]: Oops thanks for catching that, apologies for having to deal with the mismatch.

**Comment by craig-willis** [comment]
@ThomasThelen Are you planning to address the refresh problem?

**Comment by ThomasThelen** [comment]
@craig-willis I'll look into it today

**Comment by ThomasThelen** [comment]
I'm seeing this bahaviour on stage right now, if it's fine with both of you I'd like to fix it in another issue/PR.

**Pull request merged**